### ISSUE TITLE
Document `PostTick` and `PreTick` for Simulation

### ISSUE THREAD

**Issue by Doprez** [stride3d/stride-docs#342]
We should add a section to the Physics manual to show how to subscribe to the physics tick events in order to mimic what people are used to for `FixedUpdate`.

- This section should [get the simulation](https://doc.stride3d.net/4.2/en/api/Stride.Physics.PhysicsScriptComponentExtensions.html?q=GetSimulation)
- subscribe to both [PostTick](https://github.com/stride3d/stride/blob/4c46fdcc4da0021514fbaaea3d432095d164b9b6/sources/engine/Stride.Physics/Simulation.cs#L1139) and [PreTick](https://github.com/stride3d/stride/blob/4c46fdcc4da0021514fbaaea3d432095d164b9b6/sources/engine/Stride.Physics/Simulation.cs#L1157)
- add some comments as to how they run

Example:
```cs
using Stride.Engine;
using Stride.Physics;

public class FixedUpdate : StartupScript
{
	public override void Start()
	{
		var sim = this.GetSimulation();
		sim.PreTick += Sim_PreTick;
		sim.PostTick += Sim_PostTick;
	}

	private void Sim_PostTick(Simulation sender, float tick)
	{
		// Run before the Simulation is updated
	}

	private void Sim_PreTick(Simulation sender, float tick)
	{
		// Run after the Simulation is updated
	}
}
```

This should be a pretty simple manual page I think but we should also add it to the [Stride for Unity® developers](https://doc.stride3d.net/latest/en/manual/stride-for-unity-developers/index.html) somewhere that would mention `FixedUpdate` from Unity.

**Comment by VaclavElias** [comment]
Do we need to unsubscribe?

**Comment by Doprez** [comment]
Yes, absolutely worth mentioning maybe in the Cancel method that is overridable.
